src/app/components/Footer/Footer.tsx

```diff
@@ -33,6 +33,7 @@ export default function Footer() {
       <div
         className={classNames(
           styles['footer-phdi'],
+          'footer-phdi',
```

Comment:
Are these doing different things?

Reply:
changed the name so its more clear, using styles changes the name so targeting it with the global styles isn't possible